js/markupIndex.js

```diff
@@ -0,0 +1,616 @@
+  imgIconBtn.setAttribute("alt", "icon");
+
+  data.testimonials.content.forEach((elem) => {
+    const testimonialsUser = document.createElement("li");
+    const userPhoto = document.createElement("img");
+    const userDiscrp = document.createElement("div");
+    const userText = document.createElement("p");
+    const userName = document.createElement("p");
+
+    testimonialsUser.className = "testimonials__box";
+    userDiscrp.className = "testimonials__box_user";
+    userText.className = "testimonials__box_citation";
+    userName.className = "testimonials__box_text";
+
+    const userProf = userName.cloneNode(false);
+
+    userPhoto.setAttribute("src", elem.image);
+    userPhoto.setAttribute("alt", "user photo");
+    userPhoto.setAttribute("width", 280);
+    userText.textContent = elem.text;
+    userName.textContent = elem.name;
+    userProf.textContent = elem.profession;
+
+    userDiscrp.append(userText);
+    userDiscrp.append(userName);
+    userDiscrp.append(userProf);
+
+    testimonialsUser.append(userDiscrp);
+    testimonialsUser.append(userPhoto);
+    testimonialsBox.append(testimonialsUser);
+  });
+
+  divIconBtnLeft.append(imgIconBtn);
+  divIconBtnRight.append(imgIconBtn.cloneNode(false));
+  btnLeft.append(divIconBtnLeft);
+  btnRight.append(divIconBtnRight);
+
+  testimonialsContainer.append(btnLeft);
+  testimonialsContainer.append(testimonialsBox);
+  testimonialsContainer.append(btnRight);
+  container.append(
+    markupTitle(data.testimonials.title.primary, "title_border-testimonials")
+  );
+  container.append(testimonialsContainer);
+  section.append(container);
+
+  return section;
+};
+
+const markupContact = () => {
+  const section = document.createElement("section");
+  const container = document.createElement("div");
+  const contactContainer = document.createElement("div");
+  const stepBox = document.createElement("div");
+  const mapBox = document.createElement("div");
+  const formBox = document.createElement("div");
+  const mapTextBox = document.createElement("div");
+  const p = document.createElement("p");
+  const mapText = document.createElement("p");
+  const formText = document.createElement("p");
+  const stepTitle = document.createElement("h3");
+  const stepList = document.createElement("ul");
+  const iconBox = document.createElement("div");
+  const imgIcon = document.createElement("img");
+  const imgMap = document.createElement("img");
+  const iconHidepass = document.createElement("img");
+  const form = document.createElement("form");
+  const labelText = document.createElement("label");
+  const spanText = document.createElement("span");
+  const hidepass = document.createElement("span");
+  const hidepassIcon = document.createElement("span");
+  const hidepassText = document.createElement("span");
+  const inputText = document.createElement("input");
+  const btnForm = createButton("button-dark form_contact-us_btn");
+  const linkEmail = document.createElement("a");
```

Comment:
could you make the creation of the list of items in a more declarative way? and the same in other functions